modules/develop/pages/manage-topics/cloud-topics.adoc

```diff
@@ -0,0 +1,92 @@
+
+* config_ref:cloud_storage_enable_remote_write,false,properties/object-storage-properties[]
+* config_ref:cloud_storage_enable_remote_read,false,properties/object-storage-properties[]
+
+This ensures that newly-created Redpanda topics are Cloud Topics by default.
+
+For details, see xref:manage:tiered-storage.adoc#enable-tiered-storage-for-a-cluster[Enable Tiered Storage for a cluster].
+====
+
+- xref:manage:tiered-storage.adoc#configure-object-storage[Configure object storage].
+- You must have an Enterprise license.
+
+To check your license status:
+
+[,bash]
+----
+rpk cluster license info
+----
+
+If you intend to use Docker Compose while working with Cloud Topics, see the https://docs.docker.com/compose/install/[Docker Compose installation documentation^]. To verify you have it installed, run:
+
+[source,bash]
+----
+docker compose version
+----
+
+You should see the installed Docker Compose version.
+
+For more details about using Docker Compose with Redpanda, see the xref:get-started:quick-start.adoc[Redpanda Self-Managed Quickstart].
+
+endif::[]
+
+== Limitations
+
+Not all Redpanda features are available when using Cloud Topics (beta). The following features are not supported in this beta release:
+
+- Kafka compaction
+- Iceberg topics
+- Topic recovery
+- Remote Read Replicas
+
+IMPORTANT: You should expect to throw away any cluster running Cloud Topics (beta) after it becomes generally available (GA). Redpanda may be unable to continue using Cloud Topics created with the beta version, and you may be unable to upgrade to future versions. Also, any topic specified to be a Cloud Topic cannot be subsequently converted back to a standard Redpanda topic that uses local or Tiered Storage.
+
+== Enable Cloud Topics for your cluster
+
+You must first enable Cloud Topics for your cluster:
+
+[,bash]
+----
+rpk cluster config set unstable_beta_feature_cloud_topics_enabled=true
+Successfully updated configuration. New configuration version is 3.
+----
+
+Once enabled, create the Cloud Topic:
+
+[,bash]
+----
+rpk topic create -c cloud_topics_enabled=true <new-cloud-topic-name>
+TOPIC                    STATUS
+audit.analytics.may2025  OK
+----
+
+You can make a topic a Cloud Topic only at topic creation time.
+
+NOTE: In addition to replication, cross-AZ ingress (Producer) and egress (Consumer) traffic can also contribute substantially to cloud networking cots. When running multi-AZ clusters in general, Redpanda strongly recommends using xref:../consume-data/follower-fetching.adoc[Follower Fetching], which allows consumers to avoid crossing network zones. When possible, you can use xref:../produce-data/leader-pinning.adoc[leader pinning], which positions a topic's partition leader close to the producers, providing a similar benefit for ingress traffic. These features can add additional savings to the replication cost savings of Cloud Topics.
```

Comment:
```suggestion
NOTE: In addition to replication, cross-AZ ingress (producer) and egress (consumer) traffic can also contribute substantially to cloud networking cost. When running multi-AZ clusters in general, Redpanda strongly recommends using xref:../consume-data/follower-fetching.adoc[Follower Fetching], which allows consumers to avoid crossing network zones. When possible, you can use xref:../produce-data/leader-pinning.adoc[leader pinning], which positions a topic's partition leader close to the producers, providing a similar benefit for ingress traffic. These features can add additional savings to the replication cost savings of Cloud Topics.
```